docs/cow-protocol/reference/apis/driver.mdx

```diff
@@ -0,0 +1,9 @@
+---
+sidebar_position: 2
```

Comment:
nit: can we have this below orderbook?
<img width="314" alt="image" src="https://github.com/cowprotocol/docs-v2/assets/1200333/e4edae93-87dc-4353-aa30-3f66cb00f60a">